tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -459,6 +460,47 @@ public function test_next_tag_should_stop_on_closers_only_when_requested() {
 			'Did not stop at desired tag closer'
 		);
 		$this->assertTrue( $p->is_tag_closer(), 'Indicated a tag closer is a tag opener' );
+
+		$p = new WP_HTML_Tag_Processor( '<div>' );
+		$this->assertTrue( $p->next_tag( array( 'tag_closers' => 'visit' ) ), "Did not find a tag opener when tag_closers was set to 'visit'" );
+		$this->assertFalse( $p->next_tag( array( 'tag_closers' => 'visit' ) ), "Found a closer where there wasn't one" );
+	}
+
+	/**
+	 * @ticket 57852
+	 *
+	 * @covers WP_HTML_Tag_Processor::next_tag
+	 * @covers WP_HTML_Tag_Processor::is_tag_closer
+	 *
+	 * @param string $tag_name The name of the tag to test.
```

Comment:
```suggestion
```

The data provider was removed. Removing the param that was previously passed from it.